src/components/cuda_pw/cuda_common.h

```diff
@@ -0,0 +1,94 @@
+/*
+    Load cuda symbols common to perfworks and events API
+*/
+#ifndef __CUDA_COMMON_H__
+#define __CUDA_COMMON_H__
+
+#include <stdio.h>
+#include "debug_comp.h"
+#include "api_cuda.h"
+
+char* PAPI_CUDA_ROOT_ENV;
+
+void *dl1, *dl2, *dl3;
+
+#define DECLARECUDAFUNC(funcname, funcsig) CUresult ( *funcname##Ptr ) funcsig;
```

Comment:
I don't like these macros (e.g. `DECLARECUDAFUNC`) they actually make the code less readable. Can you just write the all thing instead? Can you also replace the `Ptr` at the end with `_ptr` or `_p` (avoid mix upper and lower case).

Reply:
true keep Ptr at the end